project/project.go

```diff
@@ -183,11 +180,45 @@ func (p *Project) load(file string, bytes []byte) error {
 		if err != nil {
 			return err
 		}
+		networkNames = append(networkNames, name)
+	}
+
+	for name, config := range serviceConfigs {
+		// Handle NetworkMode
+		// FIXME(vdemeester) not sure it's enough, this might need some rework (here, in service and container)
+		if config.NetworkMode == "" {
+			if p.isNetworkEnabled() {
+				config.NetworkMode = networkNames[0]
```

Comment:
I'm a little confused with these couple of lines. It looks like a default `NetworkMode` for v2 services is being set based on `networkNames[0]`, but shouldn't `networkNames` be empty if there aren't any network configurations?